### PR TITLE
[#5517] feat(auth): Paimon catalog supports Ranger plugin

### PR DESCRIPTION
**Pull request by jerqi**
### What changes were proposed in this pull request?

Paimon catalog supports Ranger plugin. Kyuubi authz plugin doesn't support to update or delete the table.

### Why are the changes needed?

Fix: #5517 

### Does this PR introduce _any_ user-facing change?

No.

### How was this patch tested?

Add IT.

### CODE REVIEW

**Comment by jerqi** [comment]
Paimon plugin doesn't support update and delete operations. So I will pend this pull request.

File: authorizations/authorization-ranger/src/test/java/org/apache/gravitino/authorization/ranger/integration/test/RangerPaimonE2EIT.java [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+            "thrift://%s:%d",
+            containerSuite.getHiveRangerContainer().getContainerIpAddress(),
+            HiveContainer.HIVE_METASTORE_PORT);
+
+    generateRangerSparkSecurityXML();
+
+    sparkSession =
+        SparkSession.builder()
+            .master("local[1]")
+            .appName("Ranger Hive E2E integration test")
```
Comment (xunliu) [inline authorizations/authorization-ranger/src/test/java/org/apache/gravitino/authorization/ranger/integration/test/RangerPaimonE2EIT.java:92]: Please update `Ranger Hive E2E integration test`

File: authorizations/authorization-ranger/src/test/java/org/apache/gravitino/authorization/ranger/integration/test/RangerPaimonE2EIT.java [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+    // Kyuubi Paimon Ranger plugin doesn't support to delete yet.
+  }
+
+  @Override
+  protected void checkDeleteSQLWithWritePrivileges() {
+    // Kyuubi Paimon Ranger plugin doesn't support to delete yet.
+  }
+
+  @Override
+  protected void checkHaveNoPrivileges() {
```
Comment (xunliu) [inline authorizations/authorization-ranger/src/test/java/org/apache/gravitino/authorization/ranger/integration/test/RangerPaimonE2EIT.java:168]: Change function name to `checkWithoutPrivileges()`?

Comment (jerqi) [reply authorizations/authorization-ranger/src/test/java/org/apache/gravitino/authorization/ranger/integration/test/RangerPaimonE2EIT.java:168]: OK.

File: authorizations/authorization-ranger/src/test/java/org/apache/gravitino/authorization/ranger/integration/test/RangerPaimonE2EIT.java (hunk 1 above)
Comment (jerqi) [reply authorizations/authorization-ranger/src/test/java/org/apache/gravitino/authorization/ranger/integration/test/RangerPaimonE2EIT.java:92]: OK. Changed to Paimon.

**Review by xunliu — APPROVED** [review]
LGTM

**Pull request merged**